src/core/simulation_sequence.jl

```diff
@@ -134,6 +134,14 @@ function _check_chronology_consistency(
     #TODO: Add more consistency checks
     return
 end
+
+function _check_step_interval_consistency(step_resolution::Dates.TimePeriod, first_stage_interval::Dates.TimePeriod)
+    if step_resolution != first_stage_interval
+        throw(IS.ConflictingInputsError("Currently specifing a step resolution not equal to the first stage interval is not supported"))
```

Comment:
"step_resolution != first_stage_interval : this specification is not currently supported, adjust step_resolution or intervals arguments."

FWIW, I still think we should drop `step_resolution`